<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>974</width>
    <height>842</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>974</width>
    <height>842</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>20</y>
      <width>541</width>
      <height>711</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="Graph_layout"/>
   </widget>
   <widget class="QPushButton" name="OK_PB">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>200</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>OK</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Export_PB">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>EXPORT</string>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>100</y>
      <width>95</width>
      <height>65</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Add_Point_PB">
       <property name="text">
        <string>ADD</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Remove_Point_PB">
       <property name="text">
        <string>REMOVE</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>13</y>
      <width>251</width>
      <height>775</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QLabel" name="Index_1_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Index 1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="Index_1_radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Beat_rate_1_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Beat Rate</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Beat_rate_1_lineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Duty_cycle_1_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Duty_cycle_1_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Duty Cycle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CC_1_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CC_1_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CC</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="IMPORT_1_PB">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (1).png</normaloff>MicrosoftTeams-image (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="SAVE_1_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (2).png</normaloff>MicrosoftTeams-image (2).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="EXPORT_1_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Share_03.png</normaloff>Share_03.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="Index_2_radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Duty_cycle_2_lineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Index_2_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Index 2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CC_2_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CC</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Beat_rate_2_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Beat Rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CC_2_lineEdit"/>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="Beat_rate_2_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Duty_cycle_2_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Duty Cycle</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="IMPORT_2_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (1).png</normaloff>MicrosoftTeams-image (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="SAVE_2_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (2).png</normaloff>MicrosoftTeams-image (2).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="EXPORT_2_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Share_03.png</normaloff>Share_03.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QLineEdit" name="Beat_rate_3_lineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Beat_rate_3_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Beat Rate</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Duty_cycle_3_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Duty Cycle</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CC_3_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CC</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Index_3_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Index 3</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CC_3_lineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Duty_cycle_3_lineEdit"/>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="Index_3_radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="IMPORT_3_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (1).png</normaloff>MicrosoftTeams-image (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="SAVE_3_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (2).png</normaloff>MicrosoftTeams-image (2).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="EXPORT_3_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Share_03.png</normaloff>Share_03.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="1">
        <widget class="QLineEdit" name="Beat_rate_4_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CC_4_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CC</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Beat_rate_4_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Beat Rate</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="Index_4_radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CC_4_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Duty_cycle_4_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Duty Cycle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Index_4_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Index 4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Duty_cycle_4_lineEdit"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="IMPORT_4_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (1).png</normaloff>MicrosoftTeams-image (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="SAVE_4_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (2).png</normaloff>MicrosoftTeams-image (2).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="EXPORT_4_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Share_03.png</normaloff>Share_03.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="1" column="1">
        <widget class="QLineEdit" name="Beat_rate_5_lineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="CC_5_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>CC</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="Beat_rate_5_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Beat Rate</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="CC_5_lineEdit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="Duty_cycle_5_lineEdit"/>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="Index_5_label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Index 5</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="Duty_cycle_1_label_">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Duty Cycle</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="Index_5_radioButton">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="IMPORT_5_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (1).png</normaloff>MicrosoftTeams-image (1).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="2">
        <widget class="QPushButton" name="SAVE_5_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>MicrosoftTeams-image (2).png</normaloff>MicrosoftTeams-image (2).png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QPushButton" name="EXPORT_5_PB">
         <property name="text">
          <string/>
         </property>
         <property name="icon">
          <iconset>
           <normaloff>Share_03.png</normaloff>Share_03.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>25</width>
           <height>25</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>974</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>Beat_rate_1_lineEdit</tabstop>
  <tabstop>CC_1_lineEdit</tabstop>
  <tabstop>Duty_cycle_1_lineEdit</tabstop>
  <tabstop>Beat_rate_2_lineEdit</tabstop>
  <tabstop>CC_2_lineEdit</tabstop>
  <tabstop>Duty_cycle_2_lineEdit</tabstop>
  <tabstop>Beat_rate_3_lineEdit</tabstop>
  <tabstop>CC_3_lineEdit</tabstop>
  <tabstop>Duty_cycle_3_lineEdit</tabstop>
  <tabstop>Beat_rate_4_lineEdit</tabstop>
  <tabstop>CC_4_lineEdit</tabstop>
  <tabstop>Duty_cycle_4_lineEdit</tabstop>
  <tabstop>Beat_rate_5_lineEdit</tabstop>
  <tabstop>CC_5_lineEdit</tabstop>
  <tabstop>Duty_cycle_5_lineEdit</tabstop>
  <tabstop>OK_PB</tabstop>
  <tabstop>Export_PB</tabstop>
  <tabstop>Add_Point_PB</tabstop>
  <tabstop>Remove_Point_PB</tabstop>
  <tabstop>Index_1_radioButton</tabstop>
  <tabstop>Index_2_radioButton</tabstop>
  <tabstop>Index_3_radioButton</tabstop>
  <tabstop>Index_4_radioButton</tabstop>
  <tabstop>Index_5_radioButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
